code/modules/admin/verbs/debug.dm

```diff
@@ -414,26 +414,20 @@ ADMIN_VERB_ADD(/client/proc/cmd_admin_dress, R_FUN, FALSE)
 		"soviet admiral",
 		"tunnel clown",
 		"masked killer",
-		"assassin",
-		"death commando",
-		"syndicate commando",
-		"special ops officer",
-		"blue wizard",
-		"red wizard",
-		"marisa wizard",
-		"nanotrasen representative",
-		"nanotrasen officer",
-		"nanotrasen captain"
+		"special ops officer"
 		)
 	var/dresscode = input("Select dress for [M]", "Robust quick dress shop") as null|anything in dresspacks
 	if (isnull(dresscode))
 		return
 
 	for (var/obj/item/I in M)
-		if (istype(I, /obj/item/weapon/implant))
+		if (istype(I, /obj/item/weapon/implant) || istype(I, /obj/item/organ))
 			continue
-		M.drop_from_inventory(I)
-		if(I.loc != M)
+
+		if (!I.get_equip_slot())
+			continue
```

Comment:
Wouldn't it better to loop through `M.get_equipped_items()` instead loop through `M.contents` and check each item with `istype(.., implant)`, `istype(.., organ)` and `get_equip_slot(..)`?
Also now organs/implants in hands (maybe ears slot too) will not be dropped.